packages/core/src/exposed-thing.ts

```diff
@@ -369,7 +369,7 @@ export default class ExposedThing extends TD.Thing implements WoT.ExposedThing {
                     new InteractionOutput(inputContent, form, this.actions[name].input),
                     options
                 );
-                if (result) {
+                if (result !== undefined) {
```

Comment:
We probably need to scan the whole codebase for cases like these at some point – IIRC, there are a number of places where we check for "falsy" values instead of `undefined` or `null`. I think there is the possibility to make the eslint rules a bit stricter in this regard to help us with that.

Reply:
I would say that sometimes in fine to use the simpler `if` when we are expecting an object, but yeah a more stricter check might be good to spot problems like the one above. Let's create a tracking issue.